Main.cpp

```diff
@@ -2471,23 +2498,30 @@ bool HandleMainArgs(int argc, char *argv[], int &appExitVar)
     // Default program return var is fail
     appExitVar = 2;
 
-	if (argc > 2)
-	{
-		for (int i = 1; i < argc; i++)
-		{
-			if (strcmp(argv[i], "-server") == 0 && i + 1 < argc)
-			{
-				std::string port = argv[i + 1];
-				g_NetworkServer.EnableServerMode();
-				g_NetworkServer.SetServerPort(port);
-			}
-
-			if (strcmp(argv[i], "-module") == 0 && i + 1 < argc)
-			{
-				g_LoadSingleModule = argv[i + 1];
-			}
-		}
-	}
+    if (argc >= 2)
+    {
+        for (int i = 1; i < argc; i++)
+        {
+            // Print loading screen console to cout
+            if (strcmp(argv[1], "-cout") == 0)
```

Comment:
@basxto Shouldn't this be argv[i]? Or is there some reason you only want it to work if cout is the first argument?